sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -281,14 +272,13 @@ internal static InterestRateSwap CreateSwapByNamedConventions(DateTimeOffset? st
             return irs;
         }
 
-        // Valid types are "Vanilla"
         internal static InterestRateSwap CreateExampleInterestRateSwap(IRSTypes type)
```

Comment:
Not your code but I think I'd argue `InterestRateSwapType` (not abbrev'd and not plural)

Reply:
done :shrug: